### ISSUE TITLE
(3/18 11:45) expected 'ACCOUNT_ID_DOES_NOT_EXIST', got 'INVALID_ACCOUNT_ID'

### ISSUE THREAD

**Issue by kimbor** [hashgraph/hedera-services#12320]
### Description

[Hashscan tx](https://hashscan.io/mainnet/transaction/1710762432.182709579)
```
---- CryptoTransfer DIFFERED (TRANSACTION_RECORD_MISMATCH) ----
Mismatched values, expected 'ACCOUNT_ID_DOES_NOT_EXIST', got 'INVALID_ACCOUNT_ID' - Matching field 'status' Matching field 'receipt'  ==> expected: <ACCOUNT_ID_DOES_NOT_EXIST> but was: <INVALID_ACCOUNT_ID>

For body,
transactionID {
  transactionValidStart {
    seconds: 1710762418
    nanos: 975925693
  }
  accountID {
    accountNum: 1025896
  }
}
nodeAccountID {
  accountNum: 10
}
transactionFee: 10000000000
transactionValidDuration {
  seconds: 120
}
cryptoTransfer {
  transfers {
  }
  tokenTransfers {
    token {
      tokenNum: 4270178
    }
    nftTransfers {
      senderAccountID {
        accountNum: 1025896
      }
      receiverAccountID {
        accountNum: 5026251
      }
      serialNumber: 11212
    }
  }
}
➡️  Expected Receipt status: ACCOUNT_ID_DOES_NOT_EXIST
exchangeRate {
  currentRate {
    hbarEquiv: 30000
    centEquiv: 337342
    expirationTime {
      seconds: 1710763200
    }
  }
  nextRate {
    hbarEquiv: 30000
    centEquiv: 338784
    expirationTime {
      seconds: 1710766800
    }
  }
}
 but was status: INVALID_ACCOUNT_ID
exchangeRate {
  currentRate {
    hbarEquiv: 30000
    centEquiv: 337342
    expirationTime {
      seconds: 1710763200
    }
  }
  nextRate {
    hbarEquiv: 30000
    centEquiv: 338784
    expirationTime {
      seconds: 1710766800
    }
  }
}
```

### Steps to reproduce

Diff testing of round 165623030

### Hedera network

other

### Version

v0.48

### Operating system

None

**Comment by kimbor** [comment]
Possibly related to issue #11885

**Comment by kimbor** [comment]
Appears fixed in `release/0.49` commit hash `ca4abc97dc`
